pyaedt/edb_core/ipc2581/ecad/cad_data/step.py

```diff
@@ -204,6 +208,15 @@ def add_layer_feature(self, layer, polys):  # pragma no cover
                 layer_feature.add_feature(poly)
         self._ipc.ecad.cad_data.cad_data_step.layer_features.append(layer_feature)
 
+    @pyaedt_function_handler()
+    def add_profile(self, poly):
```

Comment:
You are just missing some doctring for the method.

Reply:
OK thanks